### DISCUSSION TITLE
Could not find 'Cargo.toml'

### DISCUSSION THREAD

**Discussion by newfuture33** [shadowsocks/shadowsocks-android#2922] [Q&A]
When build code, get this issue. Please help me.
![image](https://user-images.githubusercontent.com/34529419/187274597-daabb510-5b01-4081-8d51-cf154418fffd.png)

**Comment by newfuture33** [comment]
Hi, everyone, please help me

**Comment by madeye** [comment]
make sure you fetch the git submodule correctly.
